hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/BootstrapConfig.java

```diff
@@ -16,28 +16,36 @@
 
 package com.hedera.node.config.data;
 
+import com.hedera.node.config.NetworkProperty;
 import com.hedera.pbj.runtime.io.buffer.Bytes;
 import com.swirlds.config.api.ConfigData;
 import com.swirlds.config.api.ConfigProperty;
 
 @ConfigData("bootstrap")
 public record BootstrapConfig(
-        @ConfigProperty(value = "feeSchedulesJson.resource", defaultValue = "feeSchedules.json")
+        @ConfigProperty(value = "feeSchedulesJson.resource", defaultValue = "feeSchedules.json") @NetworkProperty
                 String feeSchedulesJsonResource,
         @ConfigProperty(
                         value = "genesisPublicKey",
                         defaultValue = "0aa8e21064c61eab86e2a9c164565b4e7a9a4146106e0a6cd03a8c395a110e92")
+                @NetworkProperty
                 Bytes genesisPublicKey,
         @ConfigProperty(value = "hapiPermissions.path", defaultValue = "data/config/api-permission.properties")
+                @NetworkProperty
                 String hapiPermissionsPath,
         @ConfigProperty(value = "networkProperties.path", defaultValue = "data/config/application.properties")
+                @NetworkProperty
                 String networkPropertiesPath,
```

Comment:
These two are file paths, which I suspect need to be node-specific.

Reply:
In mono-service, `hapiPermissions.path` and `networkProperties.path` point to the initial contents of files `0.0.122` and `0.0.121`, respectively.

But since each property now has a `defaultValue` in its `@ConfigProperty`, do we even need them?

Reply:
Changed them to node properties.